tests/data/bfkDet.yaml

```diff
@@ -0,0 +1,75 @@
+AMP:
+  C00: {}
+  C01: {}
+  C02: {}
+  C03: {}
+  C04: {}
+  C05: {}
+  C06: {}
+  C07: {}
+  C10: {}
+  C11: {}
+  C12: {}
+  C13: {}
+  C14: {}
+  C15: {}
+  C16: {}
+  C17: {}
+CATALOG:
+  BRIGHT_SLOPE: -0.8789890876985863
+  FIT_SUCCESS: true
+  MAGNITUDES:
+  - -8.852333114100896
+  - -9.322489841066023
+  - -13.184586982587202
+  - -8.635259072241642
+  - -10.243470463835365
+  - -11.50253076347839
+  - -10.326693306928767
+  - -11.928303887578235
+  - -10.001617608492202
+  - -9.01636006707388
+  - -12.268381513169228
+  - -9.262639272036727
+  - -9.872401866991718
+  - -12.524850882914071
+  - -5.6833164172243755
+  - -8.768544056471406
+  - -10.341225414260292
+  - -8.858106359058388
+  - -8.93306446747915
+  - -8.777269141371395
+  - -12.154781876678193
+  - -11.286575171837608
+  NUM_MATCHES: 22
+  SIZE_DIFF:
+  - .nan
```

Comment:
Why are there nans in here?  And how do those compare?

Reply:
This is in the difference in size between the brighter-fatter corrected and uncorrected measurements.  A look at the associated magnitudes show that these are on the bright end, so it's likely a failure to measure the size in one (both?) of those (there's no check).  I also see that `cp_verify` is still using SDSS sizes, and I thought those had been deprecated in favor of the HSM sizes, so there's at least two issues to resolve there.  https://jira.lsstcorp.org/browse/DM-42524 will sort out these issues.

Reply:
I forgot to add: nan values in either the as-run or as-expected results are excluded from comparisons.  This is one of the workarounds for the stability issue.